source/guide/extend-android.md

```diff
@@ -88,131 +81,149 @@ At the javascript side, call the module with javascript function to receive call
 event.openURL("http://www.github.com",function(resp){ console.log(resp.result); });
 ```
 
-### Component extend
-
-There are label, image, div, scroll, ect. components in weex, you can also customize your own components.
-
-#### Step to customize a component
+## Component extend
 
 1. Customize components must extend WXComponent or WXContainer
-2. @WXComponentProp(name=value(value is attr or style of dsl)) for it be recognized by weex SDK.
+2. With the `@WXComponentProp(name = value(value is attr or style))`annotation to automatically update the attribute or style for it be recognized by weex SDK.
```

Comment:
Use the `@WXComponentProp(name = value(value is attr or style))` annotation to let the update of attribute or style be recognized automatically.